hooks/tk-multi-publish2/basic/collector.py

```diff
@@ -53,8 +53,10 @@ def process_current_session(self, parent_item):
             )
 
             self.collect_playblasts(item, project_root)
-            self.collect_alembic_caches(item, project_root)
-
+            if cmds.ls(geometry=True, noIntermediate=True):
```

Comment:
I think we want to enable session geom export even if `project_root` is not set. We only require that for the zero config workflow because it gives us a root folder to search for exported files in. I think we can create this "scene geometry" item no matter what, assuming there is geom in the scene. I suggest moving the check for scene geom and collection to line `72` after the big conditional. We'll probably end up putting other stuff there too like camera collection.